espflash/src/cli/mod.rs

```diff
@@ -133,6 +133,9 @@ pub struct FlashArgs {
     /// Logging format.
     #[arg(long, short = 'L', default_value = "serial", requires = "monitor")]
     pub log_format: LogFormat,
+    /// Minimum chip revision supported by image, in format: major * 100 + minor
+    #[arg(long, default_value = "0")]
+    pub min_chip_rev: u16,
```

Comment:
I wonder if it would be worth it to parse the `major.minor` format here (and I don't even know exactly how to do that with clap) - but might not be worth it

Reply:
Good idea! Just pushed changes to implement this in [0570570](https://github.com/esp-rs/espflash/pull/525/commits/05705702b4272003d9f28ea84ddc3866bcfc076d)